dodola/repository.py

```diff
@@ -2,142 +2,58 @@
 """
 
 import logging
-from adlfs import AzureBlobFileSystem
-from fsspec.implementations.memory import MemoryFileSystem
+from fsspec import get_mapper as fs_get_mapper
 from xarray import open_zarr
 
 
 logger = logging.getLogger(__name__)
 
 
-class _ZarrRepo:
-    """Puts a simple read-Dataset/write-Zarr interface over fsspec-like storage
-
-    This puts a basic read/write repository pattern over a file
-    system to abstract-away the data layer. Aside from the ``read()``
-    and ``write()`` methods, the additional ``get_mapper()`` method is for
-    external packages that couple directly with `fsspec` to stream output
-    immediately into storage.
-
-    This is commonly instantiated through ``adl_repository``, or for testing
-    with ``memory_repository``.
+def read(url_or_path):
+    """Read Dataset from Zarr store
```

Comment:
Ehhhhh, I'm kinda on the line about this one. In this case, I didn't originally put "zarr" because the fact that we're writing to zarr is an implementation detail that I'm trying to abstract away.

I'd prefer to keep it as is for this PR because it matches the interface we refactored from. Marking this resolved.